framework/src/media/audio/audio_manager.c

```diff
@@ -1025,9 +1023,12 @@ uint32_t get_output_bytes_to_frame(unsigned int bytes)
 	return pcm_bytes_to_frames(g_audio_out_cards[g_actual_audio_out_card_id].pcm, bytes);
 }
 
-uint16_t get_max_audio_volume(void)
+audio_manager_volume_t get_max_audio_volume(void)
 {
-	return AUDIO_DEVICE_MAX_VOLUME;
+#ifndef CONFIG_AUDIO_EXCLUDE_VOLUME
```

Comment:
This config checking added as a temp code.
It is kind of driver's config, audio manager should request through the ioctl(configure api in low-half).
return value of ioctl also should be changed.

Reply:
Okay, now all the exclude configs are removed in audio_manager.c